homeassistant/components/homekit/accessories.py

```diff
@@ -322,6 +323,14 @@ def __init__(
             CHAR_STATUS_LOW_BATTERY, value=0
         )
 
+    @property
+    def available(self):
+        """Return if accessory is available."""
+        state = self.hass.states.get(self.entity_id)
+        if state is None or state.state == STATE_UNAVAILABLE:
+            return False
+        return True
```

Comment:
```suggestion
        return state is None or state.state == STATE_UNAVAILABLE
```

Reply:
The condition in the oneliner must be inverted, mustn't it?

Reply:
Thanks, adjusted the suggestion @nikolaykasyanov